docs/Kube-API.md

```diff
@@ -292,3 +292,20 @@ spec:
 If manifests provided in configmap data section will be in bad format or configmap will not exists but will be referenced
 we will set error in Sync condition only if cluster will be ready for installation. Changing configmap should fix the issue.
 
+## Teardown procedure
+
+Deleting the ClusterDeployment will automatically trigger the deletion of its referenced AgentClusterInstall and the deletion of all the Agents connected to it.
```

Comment:
isn't this repeating what is written in lines 18, 30-31?

Reply:
Yes, I am aware.
I think it has added value to have this also here so that people reading specific section will get all the needed info.

Reply:
👍